src/fsharp/FSharp.Core/seq.fsi

```diff
@@ -223,8 +317,16 @@ namespace Microsoft.FSharp.Collections
         /// <returns>The result sequence.</returns>
         ///
         /// <exception cref="T:System.ArgumentNullException">Thrown when the input sequence is null.</exception>
+        ///
+        /// <example>
+        /// <code lang="fsharp">
+        ///     type Foo = { Bar: string }
+        ///     [{Bar = "a"}; {Bar = "b"}; {Bar = "a"}] |> Seq.countBy (fun foo -> foo.Bar) // evaluates to seq [("a", 2); ("b", 1)]
```

Comment:
In cases where the lines go about 60 chars, please put the `evaluates to` outside the `<code>` section , using `<c>...</c>` for inline code. If necessary use multiple `<example>`.   See #12134 for some examples.

Reply:
Solved.